chart/templates/_helpers.yaml

```
@@ -638,7 +638,7 @@ Create the name of the cleanup service account to use
   - -c
   - |
     CONNECTION_CHECK_MAX_COUNT=0 AIRFLOW__LOGGING__LOGGING_LEVEL=ERROR exec /entrypoint \
-    airflow jobs check --job-type SchedulerJob --hostname $(hostname)
+    airflow jobs check --job-type SchedulerJob
```

```suggestion
    airflow jobs check --job-type SchedulerJob --use-hostname-callback
```

This also needs to be done conditionally based on `airflowVersion`.

---

True thank-you :+1:

---

Don't forget, this won't be released until 2.4.0, so you'll need to make using that over `--hostname` conditional.